ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -936,22 +1051,6 @@ classes: !ZenPackSpec
     dynamicview_relations:
       impacted_by: [hostedOn]
       impacts: [orgComponent]
-  NovaApi:
```

Comment:
As I mentioned before, why is this changing?   It is not a service (it doesn't show in the nova service list, and can't be monitored in that way, and probably has different impact relationships) -  it is its own kind of software component.